test/run-check-status.bats

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bats
+
+load test_helper
+
+# Positive assertions: each of these should succeed
+@test "basic return-code checking" {
+  run      true
```

Comment:
I think `run false` would be in order too, just to show that "normal" `run` suppresses the return code.

Reply:
Good point. Added, but elsewhere.